chsdi/templates/htmlpopup/seilbahnenbundeskonzession.mako

```diff
@@ -0,0 +1,18 @@
+<%def name="anlage()">
+  <%
+  layer = 'ch.bav.seilbahnen-bundeskonzession.'
+  %>
+  <tr><td class="cell-left">${_(layer + 'anlagenr')}</td> <td>${c['attributes']['anlagenr'] or '-'}</td></tr>
+  <tr><td class="cell-left">${_(layer + 'anlagename')}</td> <td>${c['attributes']['anlagename'] or '-'}</td></tr>
+</%def>
+
+<%def name="betreiber()">
+  <%
+  layer = 'ch.bav.seilbahnen-bundeskonzession.'
+  %>
+  <tr><td class="cell-left">${_(layer + 'betreiber_tuabkuerzung')}</td> <td>${c['attributes']['betreiber_tuabkuerzung'] or '-'}</td></tr>
+</%def>
+
+<%def name="layer(translation)">
```

Comment:
In general it's a bad practice to name an attribute and a function the same in python as it gets overloaded.

Reply:
Thanks for the hint. Actually I did not want to use the variable any more, just the function.  I just forgot to change the code of this make. Good catch!